code/modules/mob/living/simple_animal/abnormality/aleph/mountain.dm

```diff
@@ -5,20 +5,21 @@
 	icon_state = "mosb"
 	icon_living = "mosb"
 	icon_dead = "mosb_dead"
-	maxHealth = 1000
-	health = 1000
+	maxHealth = 1500
+	health = 1500
 	pixel_x = -16
 	base_pixel_x = -16
 	damage_coeff = list(BRUTE = 1, RED_DAMAGE = 1.2, WHITE_DAMAGE = 0.8, BLACK_DAMAGE = 0.8, PALE_DAMAGE = 0.5)
-	melee_damage_lower = 20
-	melee_damage_upper = 30
+	melee_damage_lower = 25
+	melee_damage_upper = 35
 	melee_damage_type = RED_DAMAGE
 	armortype = RED_DAMAGE
 	rapid_melee = 2
 	stat_attack = DEAD
 	ranged = TRUE
 	speed = 2
 	move_to_delay = 2
+	generic_canpass = FALSE
```

Comment:
I'm so mad I didn't know this was a variable

Reply:
It's a cryptic variable that causes normal proc to call another one.